src/systemd/sd-ndisc.h

```diff
@@ -42,7 +42,8 @@ enum {
         SD_NDISC_OPTION_RDNSS              = 25,
         SD_NDISC_OPTION_FLAGS_EXTENSION    = 26,
         SD_NDISC_OPTION_DNSSL              = 31,
-        SD_NDISC_OPTION_CAPTIVE_PORTAL     = 37
+        SD_NDISC_OPTION_CAPTIVE_PORTAL     = 37,
+        SD_NDISC_OPTION_PREF64             = 38
```

Comment:
Hm? Why did you drop the trailing comma?

Reply:
CI failed

Reply:
yes, `,` end of enum

Reply:
I guess the failure comes from the wrong `memcmp()` below. You already fixed that, I guess we can add the trailing comma now. Please try again.

Reply:
```28/1205 systemd:headers / cc-sd-ndisc.h_c-std=iso9899_1990               FAIL             0.02s   exit status 1
>>> MALLOC_PERTURB_=87 /usr/bin/env cc -c -x c -std=iso9899:1990 -Wall -Wextra -Werror -pedantic -Wno-long-long -Wno-variadic-macros -include /work/src/src/systemd/sd-ndisc.h -o/dev/null /dev/null
――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――
stderr:
In file included from <command-line>:
/work/src/src/systemd/sd-ndisc.h:46:48: error: comma at end of enumerator list [-Werror=pedantic]
   46 |         SD_NDISC_OPTION_PREF64             = 38,
      |                                                ^
/work/src/src/systemd/sd-ndisc.h:53:40: error: comma at end of enumerator list [-Werror=pedantic]
   53 |         SD_NDISC_PREFERENCE_HIGH   = 1U,
      |                                        ^
cc1: all warnings being treated as errors
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

 329/1205 systemd:headers / cc-sd-ndisc.h_c-std=iso9899_2011               OK               0.02s
 330/1205 systemd:headers / cc-sd-ndisc.h_c-std=iso9899_2017               OK               0.03s
 331/1205 systemd:headers / cc-sd-ndisc.h_c-std=c2x                        OK               0.02s
 332/1205 systemd:headers / cc-sd-ndisc.h_c++                              OK               0.04s
 333/1205 systemd:headers / cc-sd-ndisc.h_c++-std=c++98                    FAIL             0.04s   exit status 1
>>> MALLOC_PERTURB_=25 /usr/bin/env cc -c -x c++ -std=c++98 -Wall -Wextra -Werror -pedantic -Wno-long-long -Wno-variadic-macros -include /work/src/src/systemd/sd-ndisc.h -o/dev/null /dev/null
――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――
stderr:
In file included from <command-line>:
/work/src/src/systemd/sd-ndisc.h:46:48: error: comma at end of enumerator list [-Werror=pedantic]
   46 |         SD_NDISC_OPTION_PREF64             = 38,
      |                                                ^
/work/src/src/systemd/sd-ndisc.h:53:40: error: comma at end of enumerator list [-Werror=pedantic]
   53 |         SD_NDISC_PREFERENCE_HIGH   = 1U,
      |                                        ^
cc1plus: all warnings being treated as errors``` 

https://github.com/systemd/systemd/actions/runs/6072515676/job/16472634502#step:7:6534

Reply:
these are public headers, and we compile-test them on several arcane combinations of toolchains, and on some of these you can't have trailing commas in enums

Reply:
Interesting. Thanks. Let's keep the lines as is.